mitosheet/src/mito/components/toolbar/GraphTabs/ChangeChartTypeButton.tsx

```diff
@@ -41,6 +45,30 @@ export const GRAPHS_WITH_UNIQUE_CONFIG_OPTIONS = [...new Set([
     ...GRAPHS_THAT_HAVE_LINE_SHAPE
 ])];
 
+/**
+ * This function is used to switch the x and y axes if the graph is being set to favor a certain axis.
+ * So, if there are multiple columns in the X axis and we set primaryAxis = 'y', then the x and y axes
+ * will be switched.
+ */
+const updatePrimaryAxis = (primaryAxis: 'x' | 'y', x_axis_column_ids?: ColumnID[], y_axis_column_ids?: ColumnID[]): [ColumnID[] | undefined, ColumnID[] | undefined] => {
```

Comment:
Can we call this function `getXAndYColumnIDs()` or something like that just so it's clear that this function is not actually doing the updating of the params but instead just returning the column ids